hugo/content/_index.md

```diff
@@ -47,8 +47,8 @@ pgMonitor combines multiple open-source software packages and necessary configur
 
 ### Operating Systems
 
-- Prometheus/Alertmanager & Grafana: CentOS/RHEL 7 or greater
-- Exporters (node & postgres): CentOS/RHEL 6 or greater
+- Prometheus/Alertmanager & Grafana: CentOS/RHEL 7 or greater, Win Server 2012R2 or later
+- Exporters (node, wmi, postgres): CentOS/RHEL 6 or greater (node), Win Server 2012R2 or later (WMI & postgres)
```

Comment:
Same note here about Win vs Windows, I didn't note it earlier.

Reply:
fixed